Society for Neuroscience Abstract 2016.js

```diff
@@ -0,0 +1,446 @@
+{
+	"translatorID": "1ed36c21-de68-4ba4-a27f-d724ce258db4",
+	"label": "Society for Neuroscience Abstract 2016",
```

Comment:
If we extend the target (see below), then we should also rename the translator. Maybe something like "abstractsonline.com" or "abstractsonline.com presentations". Maybe @adam3smith can suggest here something best suitable.

Reply:
I thought about it, but I don't know how abstracts of other meetings are formatted. So at the moment, I can only guarantee Society for Neuroscience 2016 and 2017. I'm happy to change the name to whatever new name later.